src/main/java/it/pn/frontend/e2e/pages/radd/RichiestaAttiPage.java

```diff
@@ -95,6 +95,12 @@ public RichiestaAttiPage(WebDriver driver) {
     @FindBy(xpath = "//button[contains(text(),'Indietro')]")
     WebElement indietroButton;
 
+    @FindBy(xpath = "//input[contains(@value,'PG')]")
```

Comment:
inserire id

Reply:
for this element there is no Id: <input class="PrivateSwitchBase-input css-1m9pwf3" name="recipientType" type="radio" value="PG">